src/middlewares/Validate.js

```diff
@@ -67,5 +48,27 @@ class ValidateUser {
       check('startDate', 'date should be validate like in this format(YYYY-DD--MM)').isBefore(startDates)
     ];
   }
+
+  /**
+* Validate input
+* @static
+* @returns {object} errors
+*/
+  static profileUpdateRules() {
+    const d = new Date();
+    const year = d.getFullYear();
+    const month = d.getMonth();
+    const day = d.getDate();
+    const minAge = new Date(year - 16, month, day).toDateString();
+    return [
+      check('gender', 'Gender should be either Male or Female').isAlpha(),
+      check('birthDate', 'Invalid Date of Birth(format: YYYY-MM-DD && Make sure you are above 16)').isBefore(minAge),
+      check('address', 'Address should be specified').isAlphanumeric(),
+      check('imageURL', 'PRofile image URL should be valid'),
+      check('department', 'department should be valid').isAlpha(),
+      check('lineManager', 'PLease provide your line manager').isAlpha(),
+      check('bio', 'Please your bio is needed to complete your profile').isAlphanumeric()
```

Comment:
Here this validation you have set might not work for a long sentence as the Bio of anyone can be, I would suggest if you do it like:  `check('bio', 'Please ......').isLength({ min: 2 }),`